src/types.ts

```diff
@@ -49,3 +49,20 @@ export interface ActivityTrigger extends FunctionTrigger {
 export interface DurableClientInput extends FunctionInput {
     type: "durableClient";
 }
+
+/**
+ * Options object provided as an optional second argument to the `client.startNew()` method
+ */
+export interface ClientStartNewOptions {
```

Comment:
I would just call this `StartNewOptions` as this method only exists in the Client anyways, so I don't think the prefix "Client" is adding all that much. What do you think?

Reply:
How about `StartNewClientOptions`? `StartNewOptions` seems a bit vague to me, but I also am not married to the name so if you feel strongly about it I can go with that

Reply:
Sure, done :)